### PR TITLE
Upgrade to Lucene 6.2.1

### PR DESCRIPTION
**Pull request by bleskes**
Change list here: https://lucene.apache.org/core/6_2_1/changes/Changes.html#v6.2.1.api_changes

### CODE REVIEW

File: core/src/main/resources/org/elasticsearch/bootstrap/security.policy [hunk 1]
```diff
@@ -31,7 +31,7 @@ grant codeBase "${codebase.securesm-1.1.jar}" {
 //// Very special jar permissions:
 //// These are dangerous permissions that we don't want to grant to everything.
 
-grant codeBase "${codebase.lucene-core-6.2.0.jar}" {
+grant codeBase "${codebase.lucene-core-6.2.1.jar}" {
```
Comment (jasontedor) [inline core/src/main/resources/org/elasticsearch/bootstrap/security.policy]: :smile:

**Review by jasontedor — APPROVED** [review]
LGTM.

**Comment by jpountz** [comment]
The new shas look missing? Did you run `gradle updateShas`?

**Comment by jasontedor** [comment]
> 
The new shas look missing? Did you run `gradle updateShas`?

I bet he just did not add them before `git commit` (we talked about upgrading via another channel).

**Comment by bleskes** [comment]
@jpountz I did but git played tricks on me and my CI blamed me for it. Coming up.

**Comment by bleskes** [comment]
@jpountz shas are not present.

**Review by jpountz — APPROVED** [review]

**Comment by bleskes** [comment]
thx @jpountz , @jasontedor

**Pull request merged**